rpc_jobs/rpc_maas.yml

```diff
@@ -0,0 +1,43 @@
+- project:
+    name: "rpc-maas-master-xenial-premerge"
+    repo_name: "rpc-maas"
+    repo_url: "https://github.com/rcbops/rpc-maas"
+    series: "master"
+    branches:
+      - "master.*"
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+    scenario:
+      - master
+      - pike
+      - ocata
+      - newton
+      - ceph
+    action:
+      - deploy:
+          FLAVOR: "performance2-15"
```

Comment:
The original jobs use "general1-8" for all scenario's except for master which uses "performance2-15". Can this be adjusted to do the same in order to save some resourcing.

Reply:
Done.

Good spot, I don't remember doing this intentionally.